scripts/build_developer.sh

```diff
@@ -27,6 +35,11 @@ pip install torch_xla[tpu] \
   -f https://storage.googleapis.com/libtpu-wheels/index.html \
   -f https://storage.googleapis.com/libtpu-releases/index.html
 
+# Install Pallas dependencies
```

Comment:
Do we also need to mention installing torch_xla[pallas] in README?

Reply:
Yes, that's a good idea.